src/lib/sql/sql_translator.hpp

```diff
@@ -33,23 +43,20 @@ class SQLTranslator final {
   /**
    * @param use_mvcc  Whether ValidateNodes should be compiled into the plan
    */
-  explicit SQLTranslator(const UseMvcc use_mvcc);
-
-  /**
-   * @return after translate_*(), contains the ParameterIDs allocated for the placeholders in the query
-   */
-  std::vector<ParameterID> parameter_ids_of_value_placeholders() const;
+  explicit SQLTranslator(const UseMvcc use_mvcc,
+                         std::shared_ptr<std::unordered_map<std::string, std::shared_ptr<Table>>> meta_tables =
```

Comment:
Doc the parameter?

Also: Is this parameter meant to be used externally? If not, should we have a private constructor that takes this argument?

Reply:
private constructor is a good idea. done